src/Makefile.am

```diff
@@ -40,7 +40,7 @@ geodtest_LDADD = libproj.la
 
 lib_LTLIBRARIES = libproj.la
 
-libproj_la_LDFLAGS = -no-undefined -version-info 12:0:0
+libproj_la_LDFLAGS = -no-undefined -version-info 12:1:0
```

Comment:
This should normally be 13:0:1 I think. If I follow correctly the rule in GDAL's HOWTO-RELEASE, in the "adding interfaces" case
```
4) Update LIBGDAL_CURRENT/REVISION/AGE macros in GDALmake.opt.in.
   - For a release with no interface changes just bump REVISION.
   - Adding interfaces, bump CURRENT/AGE, set REVISION to 0.
   - Deleting interfaces / compatibility issues - bump CURRENT, others to zero.
```

Reply:
Yes, this is a bit easier to understand. After reading the rules in the PROJ HOWTO-RELEASE i interpret it the same way. 13.0.1 it is.